#try except: catch runtime errors
try:
    x = 5
    y = 0
    z = x/y
    print(z)
except:
    print("Error!")

print("Happy New Year!");